### ISSUE TITLE
Bug camel-jasypt integration with camel-karavan 4.6.0 is not working

### ISSUE THREAD

**Issue by jabhishek01** [apache/camel-karavan#1468]
### Describe the bug

Bug camel-jasypt integration with camel-karavan 4.6.0 is not working.

We have added camel-jasypt in our application.properties of integration project. When we added secret passkey in application.properties as well. In camel route yaml file when we try to print ENC(encrypted value) then gives error as component.jasypt not found in classpath.

1) test.camel.yaml 

- route:
    id: route-e881
    nodePrefixId: route-348
    from:
      id: from-f50e
      uri: timer
      parameters:
        timerName: test
        repeatcount: 2
      steps:
        - log:
            id: log-d10e
            message: {{ENC('encrypted value')}}
            

2) property added in application.properties :           
camel.component.jasypt.password=mypasskey      

     

### Steps to reproduce the behavior

1. Go to '...'a

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Linux

### Version

4.7.0

### Relevant log output

```shell
2024-12-16 15:18:21.276  INFO 15 --- [           main] org.apache.camel.main.ProfileConfigurer  : The application is starting with profile: dev
java.lang.IllegalArgumentException: Error configuring property: camel.component.jasypt.password because cannot find component with name jasypt. Make sure you have the component on the classpath
        at org.apache.camel.main.BaseMainSupport.lambda$doAutoConfigurationFromProperties$36(BaseMainSupport.java:2217)
        at org.apache.camel.main.MainHelper.computeProperties(MainHelper.java:415)
        at org.apache.camel.main.BaseMainSupport.doAutoConfigurationFromProperties(BaseMainSupport.java:2205)
        at org.apache.camel.main.BaseMainSupport.autoConfigurationFromProperties(BaseMainSupport.java:2168)
        at org.apache.camel.main.BaseMainSupport.autoconfigure(BaseMainSupport.java:549)
```

**Comment by davsclaus** [comment]
Add `camel-jasypt` as a dependency.

**Comment by jabhishek01** [comment]
that is added in application.properties.

**Comment by jabhishek01** [comment]
@mgubaidullin Do you have any suggestion on this issue as this is not working for us even we use camel-jasypt in application.properties file.

**Comment by mgubaidullin** [comment]
There is no such component in camel-catalog. Therefore there is no such component in Karavan
https://github.com/apache/camel/tree/main/catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/components

**Comment by davsclaus** [comment]
its fixed for camel 4.10.0 release

**Comment by jabhishek01** [comment]
@davsclaus  Okay. Is there way to make it work with camel-karavan 4.6 ?

**Comment by davsclaus** [comment]
No there is not

**Comment by davsclaus** [comment]
Fixed in Camel 4.10, and this is NOT a Karavan bug
